.travis.yml

```diff
@@ -52,11 +61,35 @@ matrix:
      sudo: true
    - env: TOXENV=py35-dj110head-postgres-noelasticsearch
      python: 3.5
+   - env: TOXENV=py35-dj111-sqlite-noelasticsearch
+     python: 3.5
+   - env: TOXENV=py35-dj111-postgres-noelasticsearch
+     python: 3.5
+   - env: TOXENV=py35-dj111-mysql-noelasticsearch
+     python: 3.5
+   - env: TOXENV=py35-dj111-postgres-elasticsearch
+     python: 3.5
+   - env: TOXENV=py35-dj111-postgres-elasticsearch2 INSTALL_ELASTICSEARCH2=yes
+     python: 3.5
+     sudo: true
+   - env: TOXENV=py35-dj111-postgres-elasticsearch5 INSTALL_ELASTICSEARCH5=yes
+     python: 3.5
+     sudo: true
   allow_failures:
+    - env: TOXENV=py27-dj110-mysql-elasticsearch5 INSTALL_ELASTICSEARCH5=yes
```

Comment:
This env was already in the allow_failures list (because Travis CI intermittently fails to start Elasticsearch, and we can do without the extra noise) - I just reordered the list here to match the one above. Clarifying comment added in 0e50d831fbdced15155db484461bcb4cb9b4097d.